recursion-schemes-in-rust/README.md

```diff
@@ -0,0 +1,5 @@
+# Practical Recursion Schemes in Rust
+
+This is an associated repository for Tweag's blog post "Practical Recursion
+Schemes in Rust", which gathers the examples from the post as well as an
```

Comment:
I would put a link in here, once you know the publication date. If we presume it to be Thursday this week:

```suggestion
This is an associated repository for Tweag's blog post ["Practical Recursion
Schemes in Rust"](https://tweag.io/blog/2025-04-10-rust-recursion-schemes), which gathers the examples from the post as well as an
```